File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p02755/p02755_num8_parsed.py
A, B = map(int, input().split())

for price in range(1, 1000):
    tax_8 = int(price * 0.08)
    tax_10 = int(price * 0.10)
    if tax_8 == A and tax_10 == B:
        print(price)
        break
else:
    print(-1)

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p02755/p02755_num8_parsed.cpp
#include <iostream>
#include <cmath>

int main() {
    int A, B;
    std::cin >> A >> B;

    bool found = false;
    int price;

    for (int p = 1; p <= 1000000; p++) {
        int tax8 = std::floor(p * 0.08);
        int tax10 = std::floor(p * 0.1);

        if (tax8 == A && tax10 == B) {
            price = p;
            found = true;
            break;
        }
    }

    if (found) {
        std::cout << price << std::endl;
    } else {
        std::cout << -1 << std::endl;
    }

    return 0;
}
